zerver/tests/test_subs.py

```diff
@@ -2435,20 +2437,20 @@ def attempt_unsubscribe_of_principal(
         if cache_count is not None:
             self.assert_length(cache_tries, cache_count)
 
-        # If the removal succeeded, then assert that Cordelia is no longer subscribed.
+        # If the removal succeeded, then assert that target users is no longer subscribed.
         if result.status_code not in [400]:
             subbed_users = self.users_subscribed_to_stream(stream_name, user_profile.realm)
             for user in target_users:
                 self.assertNotIn(user, subbed_users)
 
         return result
 
-    def test_cant_remove_others_from_stream(self) -> None:
+    def test_cant_remove_other_users_from_stream(self) -> None:
         """
-        If you're not an admin, you can't remove other people from streams.
+        If you're not an admin, you can't remove other people from streams except your own bots.
         """
         result = self.attempt_unsubscribe_of_principal(
-            query_count=5,
+            query_count=6,
```

Comment:
Why is this query count updated?
Rest all looks good to me.

Reply:
Because in this test case it checks one more `condition` now in `views/streams.py` which leads to an increase in query_count

Reply:
Yes I get it. Looks good to me